<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">
    <title>Progress - NEU-China</title>

    <!-- core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <link href="css/font-awesome.min.css" rel="stylesheet">
    <link href="css/animate.min.css" rel="stylesheet">
    <link href="css/prettyPhoto.css" rel="stylesheet">
    <link href="css/main.css" rel="stylesheet">
    <link href="css/responsive.css" rel="stylesheet">
    <!--[if lt IE 9]>
    <script src="js/html5shiv.js"></script>
    <script src="js/respond.min.js"></script>
    <![endif]-->
    <link rel="shortcut icon" href="images/__ico/favicon.ico">
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="images/__ico/apple-touch-icon-144-precomposed.png">
    <link rel="apple-touch-icon-precomposed" sizes="114x114" href="images/__ico/apple-touch-icon-114-precomposed.png">
    <link rel="apple-touch-icon-precomposed" sizes="72x72" href="images/__ico/apple-touch-icon-72-precomposed.png">
    <link rel="apple-touch-icon-precomposed" href="images/__ico/apple-touch-icon-57-precomposed.png">
</head><!--/head-->

<body>

<header id="header">
    <nav class="navbar navbar-inverse" role="banner">
        <div class="container">
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                <a class="navbar-brand" href="index.html"><span style="font-size: 26px"><span style="color: #F00;">I</span><span style="color: #0F0;">T</span><span style="color: lightblue;">S</span> COLOR</span></a>
            </div>

            <div class="collapse navbar-collapse navbar-right">
                <ul class="nav navbar-nav">
                    <li><a href="index.html">Home</a></li>
                    <li><a href="background.html">Background</a></li>
                    <li class="dropdown">
                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Project <i class="fa fa-angle-down"></i></a>
                        <ul class="dropdown-menu">
                            <li><a href="description.html">Description</a></li>
                            <li><a href="design.html">Design</a></li>
                            <li><a href="experiment-results.html">Experiment & Results</a></li>
                            <li><a href="proof.html">Proof of concept</a></li>
                        </ul>
                    </li>
                    <li class="dropdown">
                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Lab <i class="fa fa-angle-down"></i></a>
                        <ul class="dropdown-menu">
                            <li><a href="team.html">Team</a></li>
                            <li><a href="attributions.html">Attributions</a></li>
                            <li><a href="lab-life.html">Lab Life</a></li>
                        </ul>
                    </li>
                    <li class="dropdown active">
                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook <i class="fa fa-angle-down"></i></a>
                        <ul class="dropdown-menu">
                            <li><a href="progress.html">Progress</a></li>
                            <li><a href="protocol.html">Protocol</a></li>
                        </ul>
                    </li>
                    <li><a href="safety.html">Safety</a></li>

                    <li class="dropdown">
                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Parts <i class="fa fa-angle-down"></i></a>
                        <ul class="dropdown-menu">
                            <li><a href="our-parts.html">Our Parts</a></li>
                            <li><a href="biobricks-for-medal.html">Biobricks for medal</a></li>
                            <li><a href="basic-parts.html">Favorite Biobricks</a></li>
                        </ul>
                    </li>
                    <li><a href="human-practice.html">Human Practice</a></li>
                    <li><a href="collaborations.html">Collaborations</a></li>
                </ul>
            </div>
        </div><!--/.container-->
    </nav><!--/nav-->

</header><!--/header-->


<section id="blog" class="container">
    <div class="row">
        <div class="col-md-10 col-md-offset-1">
            <div class="title-img">
                <img class="img-responsive" src="images/notebook/progress-top.png" width="100%" />
                <h2 style="left: 38%; top: 5%; margin-left: -30px;">Progress</h2>
            </div>
        </div>
    </div>
    <div class="blog">
        <div class="row">
            <div class="col-md-10 col-md-offset-1">
                <div class="blog-item">
                    <div class="row">
                        <div class="col-xs-12 blog-content">
                            <div class="">

                            </div>
                            <ol>
                                <!--1-->
                                <li>
                                    <p>
                                        Extraction of the total RNA of Arabidopsis thaliana, then reverse transcription PCR to obtain the cDNA.Amplification of the CIB1 gene and CRY2 gene from the cDNA. Nest PCR to verify the length and to amplify the PCR product.
                                    </p>
                                    <p>
                                        Time:14/4/2016~21/4/2016
                                    </p>
                                    <ul>
                                        <li>
                                            Borrowed a plant of Arabidopsis thaliana from Shenyang Normal University and got ready to start the experiment.
                                        </li>
                                        <li>
                                           Extracted of the total RNA of Arabidopsis thaliana twice times. The RNA we extracted for the first time showed a little degraded. The second time, we changed the extraction kit and improved the grinding method and the gel testing was here. It was apparent that there existed 3 bands which might mean 28S、18S、5S RNA. 
                                             <p class="text-center">
                                            <img src="images/notebook/progress1.1.png" height="250">
                                            <br/>
                                            <strong>Used Plant RNA Kit to extract the total RNA.</strong>
                                            </p>

                                            <div class="text-center">
                                                <img src="images/notebook/progress1.2.png" height="250">
                                            </div>
                                    </ul>
                                    <p>
                                        Extracted the total RNA again in an improved method.There was nothing to do with the band of DNA marker. We just wanted to see how did the total RNA degrade and then we performed RT-PCR with the OligdT primer rather than random primers immediately to obtain the cDNA of mRNA.
                                    </p>
                                    <ul>
                                        <li> Amplified the CIB1 gene and CRY2 gene with designed primers from the cDNA we extracted before. Both of them showed pretty weak bands which could be hardly see. A nest PCR was performed to increase concentration and to enhance specificity. </li>
                                        <li> We extracted and purified the correct length of those two gene fragments as shown below: </li>
                                         <p class="text-center">
                                            <img src="images/notebook/progress1.3.png" height="250">
                                            <br/>
                                            <strong>A weak band in the left picture, and a clear band in the right after NEST PCR. </strong>
                                        </p>
                                        <p class="text-center">
                                            <img src="images/notebook/progress1.4.png" height="250">
                                            <br/>
                                            <strong>CRY2 gene was successfully extracted from cDNA.</strong>
                                        </p>
                                    </ul>
                                    <i> Abstract We extracted the total RNA in a normal way according the kit instruction at the first time. The second time we improved our experiment by changing grinding tools and exposing the plant under blue light overnight.Finally, We have successfully obtained the fragments of the correct length as so far. </i>
                                </li>
                                <!--2-->
                                <li>
                                    PCR amplification of the tCas9 gene from the commercial pHS-CR042 plasmid. Overlap PCR the tCas9 and CIB1 gene after gel purification respectively. Restriction digest of the overlap PCR product and the expression vector pBD024. Ligation of the digested products and then transformed the ligation product into DH5alpha bacteria cells. Patch and miniprep to obtain our expression plasmid that produces tCas9-CIBN fusion protein.
                                    <ul>
                                        <li>Amplified of the tCas9 gene from the commercial pHS-CR042 plasmid and modified at 5’ end. Overlap pcr was applied to fuse CIBN with tCas9 after purification. </li>
                                        <p class="text-center">
                                            <img src="images/notebook/progress2.1.png" height="250">
                                            <br/>
                                            <strong></strong>
                                        </p>
                                        The top strip was the overlapping product tCas9-CIBN. The other bands were by-products.
                                        <li>
                                            Gel extracted of the fusion gene and then ligated with pSB024 vector. 
                                        </li>
                                        <p class="text-center">
                                            <img src="images/notebook/progress2.2.png" height="250">
                                            <br/>
                                            <strong></strong>
                                        </p>
                                        Picked out SC(single colony)-3 and SC-4 containing tCas-CIB1 to culture overnight. Extracted the plasmid.
                                        <p class="text-center">
                                            <img src="images/notebook/progress2.3.png" height="250">
                                            <br/>
                                            <strong></strong>
                                        </p>
                                        Double enzyme (Kpn I, Xba I ) digestion test showed the correct band of tCac9-CIB1(5Kb)
                                        <li>
                                            After gene sequencing, we found that the vector we constructed appeared the frameshift mutation. We consulted the literature again, adjusted the structure of the CIB1 gene, and cloned the partial sequence of the N terminal before the mutation, namely CIBN.
                                        </li>
                                        <p class="text-center">
                                            <img src="images/notebook/progress2.4.png" height="250">
                                            <br/>
                                            <strong></strong>
                                        </p>
                                        Modified the CIB1 gene. Amplified CIBN geng by specific primers. 
                                        <li>
                                            Next, the same procedure was carried out: PCR with specific primers we booked, double enzyme digestion, phenol purification, ligation, transformation, colony PCR, path ,and restriction digest testing at last. the following is our digestion test results, sequencing results show completely consistent with the expected return sequence. 
                                        </li>
                                        <p class="text-center">
                                            <img src="images/notebook/progress2.5.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                        SC-2, SC-6, SC-9, SC-11, SC-13, SC-15 were verified correct through colony PCR.
                                        <p class="text-center">
                                            <img src="images/notebook/progress2.6.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                        Enzyme (BamH I, XbaI) cut and the length of band were correct.
                                    </ul>
                                    <br>
                                    <i>
                                        Abstract: Unexpected mutation might occur when you amplify a specific gene through PCR method. What’s more, the efficiency of restricted digestion and ligation would also effect the result of constructing a functional plasmid. We have improved our protocal according to the experiments we performed before: PCR annealing temperature was reset at 55 degrees Celsius up and down; After removal from the PCR instrument, the PCR product was rapidly cooled; Enzyme digestion for the night to ensure its efficiency; ligation and transformation were both performed on the ice with low temperature and so on. In conclusion, we have constructed an expression vector for fusion protein expression pBD024-tCas9-CIBN. The corresponding improvements were made about the the mutation occurred at experiments .
                                    </i>
                                </li>
                                <!--3-->
                                <li>
                                    PCR amplification of the synthetic VP64. Cloning the VP64 into SK plasmid which is used in our own laboratory. Sending the plasmid to sequence after digest testing. Overlap PCR of the CRY2 and VP64 gene after gel purification respectively. Restriction digest of the overlap products and a commercial plasmid pBD021 and then ligated. Transformed into DH5alpha and patched. Plasmid preparation of the plasmid after colony PCR testing. 
                                    <ul>
                                        <li> 
                                            Vp64, the transcriptional activator we purchased from the company turned out to be wrong to a certain extent. the DNA didn’t match the sequence of NCBI after BLAST. Therefore, we tried to synthesis of that gene entirely. 
                                            <p class="text-center">
                                            <img src="images/notebook/progress3.1.png" >
                                            <br/>
                                            <strong></strong>
                                            </p>
                                        </li>
                                        <li>
                                            Amplified of the synthetic VP64 and cloned it into SK plasmid. Send this plasmid to sequencing and the feedback showed that the VP64 synthesized completely was correct, It meant that we could continue to use it in the later experiment. The following picture showed the correct result of double enzyme digestion test.
                                            <p class="text-center">
                                            <img src="images/notebook/progress3.2.png" >
                                            <br/>
                                            <strong></strong>
                                            </p>
                                            <p class="text-center">
                                            <img src="images/notebook/progress3.3.png" >
                                            <br/>
                                            <strong></strong>
                                            </p>
                                        </li>
                                        <li>
                                            Amplified of the CRY2 gene we extracted at the first step. Overlap PCR to connect the CRY2 and PV64 together after gel purification respectively. 
                                        </li>
                                        <li>
                                            Gel extracted of the fusion gene and then ligated with pSB024 vector. After sending the constructed plasmid to sequencing, time had passed for nearly a week. 
                                            <p class="text-center">
                                                <img src="images/notebook/progress3.4.png" >
                                                <br/>
                                                <strong></strong>
                                            </p>
                                        </li>
                                        <li>
                                            A bad news made us mad after analyzing the feedback gave by sequencing company that there existed 3 intron fragments in CRY2 gene due to incompletely removal of genome DNA. We had no method but to clone the 4 extrons respectively and to fuse them together by the means of redesigning overlapping primers.
                                            <p class="text-center">
                                                <img src="images/notebook/progress3.5.png" >
                                                <br/>
                                                <strong></strong>
                                            </p>
                                        </li>
                                        <li>
                                            Fortunately, the progress of removing the introns went smoothly. You could see from the picture below that we purified the CRY2 without introns successfully and the sequence analyzing proved it correctly at the same time.
                                            <p class="text-center">
                                                <img src="images/notebook/progress3.6.png" >
                                                <br/>
                                                <strong></strong>
                                            </p>
                                        </li>
                                        <li>
                                            Next, the same procedure was carried out: PCR with specific primers we booked, double enzyme digestion, phenol purification, ligation, transformation, colony PCR, path ,and restriction digest testing at last. the following is our digestion test results, sequencing results showed that we had successfully constructed the pBD021-CRY2-VP64 expression vector.
                                            <p class="text-center">
                                                <img src="images/notebook/progress3.7.png" >
                                                <br/>
                                                <strong></strong>
                                            </p>
                                            <p class="text-center">
                                                <img src="images/notebook/progress3.8.png" >
                                                <br/>
                                                <strong></strong>
                                            </p>

                                        </li>
                                    </ul>
                                    <i>
                                        Abstract: prepare a backup in case of need. Overall, we successfully constructed the pBD021-CRY2-VP64 plasmid even though the process is tortuous.
                                    </i>
                                </li>
                                <!--4-->
                                <li>
                                    Transformed the pBD024-tCas9-CIBN and pBD021-CRY2-VP64 expression plasmid into BL21 E.coli strain respectively. Cultured in LBK(LB culture with kanamycin antibiotics) overnight. Protein expression induced by arabinose. Protein extraction from the E.coli culture. western blot or Silver Staining after SDS-PAGE Gel electrophoresis to detect if the protein is expressed.
                                </li>
                                <ul>
                                    <li> 
                                        Designed experiments to check if the fusion protein would be expressed in prokaryote cell such as E.coli. Transformed the pbD024-tCas9-CIBN and pBD021-CRY2-VP64 vectors into BL21 strain respectively. 
                                    </li>
                                    <li>
                                        Colony PCR was done to select positive colonies after transformation. pBD024 plasmid consisted of inducible promoter (pBad/araC). Protein expression started within the existence of arabinose. 
                                    </li>
                                    <li>
                                        Coomassie blue staining and silver staining was performed at first, but the effects were not satisfactory. The purpose band was not distinguished clearly from negative control group. Under that condition, we planed to do western blot to verify the existence of tCas9-CIBN protein’s HA tag. 
                                    </li>
                                    <li>
                                        We explored the appropriate induce condition via pre-experiment. Data proved the protein was truly expressed. We found that 4mM arabinose, 30 Celsius culture temperature, and 16h culture time were the best choice to induce the expression of protein tCas9-CIBN. 
                                    </li>
                                    <li>
                                        Protein CRY2-VP64 was also detected by western bolt rather than coomassie blue staining and silver staining. The best culture condition concluded above was applied into that experiment but did not use arabinose because of the constitutive promoter of pBD021. The results are illustrated as bellows:
                                    </li>
                                    <p class="text-center">
                                        <img src="images/notebook/progress3.8.png" >
                                        <br/>
                                        <strong></strong>
                                    </p>

                                </ul>
                                <i>
                                    Abstract: the result of western blot showed that tCas9-CIBN protein and CRY2-VP64 protein were detected. Next we would test the function of LACE system by means of co-transformation of these device vector.
                                </i>
                                <!--5-->
                                <li>
                                    gRNA designed and booked the specific primers. Cloned the gRNA fragment into the expression plasmid containing CRY2-VP64. Transformed all of the pBD024-tCas9-CIBN, pBD021-CRY2-VP64-gRNA and pCold-GFP (containing the gRNA targeted reporter gene ) into BL21 E.coli strain. Test the function of the light-inducible system in prokaryote cell after selection.
                                </li>
                                <!--6-->
                                <li>
                                    Used the methods above to construct the eukaryotic expression vector. cloned the right gene into PESC and P426 eukaryotic expression vector respectively. Send the plasmid to sequencing.
                                    <ul>
                                        <li>
                                            Cloned tCas9-CIBN, CRY2-VP64 into pESC vector, this plasmid consisted of eukaryotic promoter and we added yeast RBS at the N terminal of the gene. After send the constructed vector to sequencing, we designed the next step to transform it into yeast by some different ways.
                                        </li>
                                    </ul>
                                </li>
                                <!--7-->
                                <li>
                                    Overlap of the tCas9 and VP64 gene and cloned it to different empty expression plasmids, which acted as control.
                                </li>
                                <ul>
                                    <li>
                                        We designed a control experiment to verify the validity of the LACE system. However, we met across some trouble of creating the tCas9-VP64 fusion gene.
                                        <p class="text-center">
                                            <img src="images/notebook/progress7.1.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                        <p class="text-center">
                                            <img src="images/notebook/progress7.2.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                    </li>
                                    <li>
                                        At first, we followed the regular progress to perform overlap PCR. A unexpected phenomenon came up that gel electrophorisis showed nothing, even a weak band, which never happened before. We guessed that it might have something to do with the length of that two genes,tCas9 is about 4Kb and VP64 240bp. So, we planed to change the experimental method to create that fusion gene.
                                    </li>
                                    <li>
                                        Changed the method of purification before overlap PCR. We purified the PCR products through gel extraction. But this time, we tried phenol purification instead. The result was not better in that way. 
                                    </li>
                                    <li>
                                        Changed the origin of the VP64 amplification. We used double enzymes to digest the SK-VP64 plasmid constructed before to see if the plasmid was degraded. Finally, unclear strips let us down and we had to turn eyes to other plasmid which contained VP64. We were so upset to accept the fact that we were failed again. Colony PCR demonstrated that there existed something wrong during the previous experiments.
                                        <p class="text-center">
                                            <img src="images/notebook/progress7.3.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                        <p class="text-center">
                                            <img src="images/notebook/progress7.4.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                    </li>
                                    <li>
                                        Then we doubted the annealing temperature might influenced the result. The optimal annealing temperature for that reaction was proposed by gradient PCR(51 Celsius ~59 Celsius). We could conclude from the picture that every temperature between 51 Celsius to 59 Celsius was okay to overlap contrast to another gene’s obvious changes of gradient on the left. We were confronted with a series weak bands full of confusion. What we chose to do next was to continue base on the unsatisfactory result. 
                                        <p class="text-center">
                                            <img src="images/notebook/progress7.5.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                    </li>
                                    <li>
                                        When we cloned the fusion gene into eukaryote vector pESC, we found there existed a correct strip via colony PCR. Then, we extracted the plasmid from the right colony. After enzyme digestion test and sequencing test, we finally attained the right control vector.
                                        <p class="text-center">
                                            <img src="images/notebook/progress7.6.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                    </li>
                                    <li>
                                        Amplified the prokaryote fragment of tCas9-VP64 from the eukaryote vector we constructed above. 
                                        <p class="text-center">
                                            <img src="images/notebook/progress7.7.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                    </li>
                                </ul>
                                <!--8-->
                                <li>
                                    Parts preparation. All parts for the submission are amplified using PCR that added the biobrick restriction sites. After the restriction digest, these fragments were cloned into the PSB1C3 vectors. Finally, the parts were verified using colony PCR and send to sequencing.
                                    <ul>
                                        <li>
                                            At first, we extracted the pSB1C3 backbone from the kit. Then selected the red colonies which expressed RFP to culture and extracted the plasmid. 
                                        </li>
                                        <li>
                                            Amplifed the fragments that would be submitted: tCas9, tCas9-CIBN, CIBN, CRY2, VP64, CRY2-VP64. All of these parts consisted of prokaryotic RBS.
                                        </li>
                                        <p class="text-center">
                                            <img src="images/notebook/progress8.1.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                        <p class="text-center">
                                            <img src="images/notebook/progress8.2.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                        <p class="text-center">
                                            <img src="images/notebook/progress8.3.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                        <p class="text-center">
                                            <img src="images/notebook/progress8.4.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                        <p class="text-center">
                                            <img src="images/notebook/progress8.5.png" >
                                            <br/>
                                            <strong></strong>
                                        </p>
                                    </ul>
                                </li>
                                <!--9-->
                                <li>
                                    Molecular cloning of the part ..... Improved our plasmid....
                                </li>
                            </ol>
                        </div>
                    </div>
                </div><!--/.blog-item-->
            </div><!--/.col-md-10-->

        </div><!--/.row-->

    </div><!--/.blog-->

</section><!--/#blog-->


<section id="bottom">
    <!--LOGO群-->
    <div class="footer-logo text-center wow fadeInDown">
        <img style="width: 105px;" src="images/footer-logo/iGEM.png"/>
        <img style="width: 175px;" src="images/footer-logo/NEU.png"/>
        <img style="width:  98px;" src="images/footer-logo/CLHS.png"/>
        <img style="width: 112px;" src="images/footer-logo/SC.png"/>
        <br/>
        <img style="width: 112px;" src="images/footer-logo/FungiDB.png"/>
        <img style="width:  98px;" src="images/footer-logo/GENEWIZ.png"/>
        <img style="width:  98px;" src="images/footer-logo/geshare.png"/>
        <img style="width: 112px;" src="images/footer-logo/IDT.png"/>
        <img style="width: 112px;" src="images/footer-logo/mathworks.png"/>
        <img style="width:  91px;" src="images/footer-logo/NCBI.png"/>
        <img style="width:  91px;" src="images/footer-logo/NEB.png"/>
        <br/>
        <img style="width: 126px;" src="images/footer-logo/OMEGA.png"/>
        <img style="width: 112px;" src="images/footer-logo/SA.png"/>
        <img style="width: 126px;" src="images/footer-logo/SanGon.png"/>
        <img style="width:  84px;" src="images/footer-logo/SMS.png"/>
        <img style="width: 112px;" src="images/footer-logo/SnapGene.png"/>
        <img style="width:  98px;" src="images/footer-logo/TaKaRa.png"/>
    </div>
</section><!--/#bottom-->

<footer id="footer" class="midnight-blue">
    <div class="container">
        <div class="row">
            <div class="col-sm-6">
                &copy; 2016 NEU-China Team.
            </div>
            <div class="col-sm-6">
                <ul class="pull-right">
                    <li><a href="index.html">Home</a></li>
                    <li><a href="team.html">Our Team</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer><!--/#footer-->

<script src="js/jquery.js"></script>
<script src="js/bootstrap.min.js"></script>
<script src="js/jquery.prettyPhoto.js"></script>
<script src="js/jquery.isotope.min.js"></script>
<script src="js/main.js"></script>
<script src="js/wow.min.js"></script>
</body>
</html>